index.html

```diff
@@ -1,29 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link
-            href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap"
-            rel="stylesheet"
-        />
-        <link rel="stylesheet" href="./styles/style.css" />
-        <link rel="stylesheet" href="./styles/reset.css" />
-        <link rel="stylesheet" href="./styles/global.css" />
-
-        <title>Cool Web</title>
-    </head>
-
-    <body>
-        <header>
-            <h1>Cool Web Project</h1>
-        </header>
-
-        <main></main>
-
-        <footer>&copy; 2021 Aspiring Software Developer</footer>
-
-        <script type="module" src="./app.js"></script>
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/global.css" />
+
+    <title>Cool Web</title>
+</head>
+
+<body>
+    <header>
+        <h1>Cool Web Project</h1>
+    </header>
+    <select name="api" id="api">
+        <option value="star-wars">Star Wars</option>
+
+        <option value="pokemon">Pokemon</option>
+    </select>
```

Comment:
Ideally, the main content of your page will reside within the `<main>` tag. You'll notice a common structure of HTML is like this: 

```js
<body>
  <header>
    My header
  </header>

  <main>
    Main content // can include anything from rendered content to different structural UI elements like select, forms, input fields
  </main>

  <footer>
    typically contains citations/copyright/reference ect info - or really whatever you want :)
  </footer>
</body>
```

This also helps to ensure that all of your content is contained within a specific parent element so that it remains in view on the page rather than being pushed off-screen slightly like the API results are being currently.